plugins/resources/compound/libcompound.cpp

```diff
@@ -1852,6 +1922,15 @@ irods::error open_for_prefer_cache_policy(
         rodsLog(LOG_NOTICE, "[%s] - operation not found in property map; using open.", __FUNCTION__);
     }
 
+    // get the replica statuses for cache and archive
+    int cache_replica_status, archive_replica_status;
+    int cache_replica_number, archive_replica_number;
```

Comment:
Please declare each variable on its own line.

These need to be initialized.

Reply:
done